cmd/internal/find/usage.go

```diff
@@ -0,0 +1,36 @@
+package find
+
+import (
+	"github.com/puppetlabs/wash/cmd/internal/find/types"
+	"github.com/puppetlabs/wash/cmd/internal/find/primary"
+)
+
+// Usage returns `wash find`'s usage string
+func Usage() string {
+	u := ""
+	u += "Recursively descends the directory tree of the specified path, evaluating an\n"
+	u += "expression composed of 'primaries' and 'operands' for each entry in the tree\n"
```

Comment:
Can you fit in a succinct section on operands? I also think quoting `expression` might help highlight the connection to Usage below.

Reply:
Done. Here's the new `usage`

```
enis:wash enis.inan$ ./wash find -h
Recursively descends the directory tree of the specified path, evaluating an
'expression' composed of 'primaries' and 'operands' for each entry in the tree.

Usage:
  wash find [path] [options] [expression]

Flags:
      -depth             Visit the children first before the parent (default false)
      -mindepth levels   Do not print entries at levels less than levels (default 0)
      -maxdepth levels   Do not print entries at levels greater than levels (default infinity)
  -h, -help              Print the usage
  -h, -help primary      Print a detailed overview of the specified primary
  -h, -help 'syntax'     Print a detailed overview of find's expression syntax

Primaries:
      -action action     Returns true if the entry supports action
      -true              Always returns true
      -false             Always returns false
  -m, -meta metaP        Returns true if the entry's meta attribute satisfies metaP
      -name glob         Returns true if the entry's cname matches glob
      -path glob         Returns true if the entry's normalized path matches glob
      -size sizeP        Returns true if the entry's size attribute satisfies sizeP
      -ctime timeP       Returns true if the entry's ctime attribute satisfies timeP
      -mtime timeP       Returns true if the entry's mtime attribute satisfies timeP
      -atime timeP       Returns true if the entry's atime attribute satisfies timeP

Operands:
  ( e )                  Parentheses operator. Evaluates to true if e evaluates to true
   !, -not e             Logical NOT operator. Evaluates to true if e evaluates to false
  e1 -and e2             Logical AND operator. Evalutes to true if both e1 and e2 are true
  e1 -a e2
  e1 e2
  e1 -or e2              Logical OR operator. Evalutes to true if either e1 or e2 are true
  e1 -o e2

Use "wash find --help <primary>" for more information about a primary. To view
a detailed overview of find's expression syntax, use "wash find --help syntax".

NOTE: All entry attribute primaries return false if the given entry does not have
the specified attribute. For example, the -mtime primary will always return false
if the entry does not have an mtime attribute.
```